set rpath for prepare_builtins build utility to find correct stage2
libs at runtime when executed as part of a build
Index: llvm-toolchain-snapshot_16~++20221123082119+16bf359a3faa/libclc/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_16~++20221123082119+16bf359a3faa.orig/libclc/CMakeLists.txt
+++ llvm-toolchain-snapshot_16~++20221123082119+16bf359a3faa/libclc/CMakeLists.txt
@@ -115,6 +115,9 @@ add_llvm_executable( prepare_builtins ut
 target_compile_definitions( prepare_builtins PRIVATE ${LLVM_VERSION_DEFINE} )
 # These were not properly reported in early LLVM and we don't need them
 target_compile_options( prepare_builtins PRIVATE -fno-rtti -fno-exceptions )
+set_target_properties( prepare_builtins
+                       PROPERTIES INSTALL_RPATH "${LLVM_LIBDIR}"
+                       BUILD_WITH_INSTALL_RPATH "ON")
 
 # Setup arch devices
 set( r600--_devices cedar cypress barts cayman )
